charts/kubewarden-controller/templates/rbac.yaml

```diff
@@ -206,6 +206,71 @@ rules:
   - create
 ---
 apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: audit-scanner-cluster-role
+  labels:
+    {{- include "kubewarden-controller.labels" . | nindent 4 }}
+  annotations:
+    {{- include "kubewarden-controller.annotations" . | nindent 4 }}
+rules:
+- apiGroups:
+  - policies.kubewarden.io
+  resources:
+  - clusteradmissionpolicies
+  - admissionpolicies
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - policies.kubewarden.io
+  resources:
+  - clusteradmissionpolicies/status
+  - admissionpolicies/status
+  verbs:
+  - get
+  - patch
+  - update
+- apiGroups:
+    - policies.kubewarden.io
+  resources:
+    - policyservers
+  verbs:
+    - get
+    - list
+    - watch
+- apiGroups:
+    - policies.kubewarden.io
+  resources:
+    - policyservers/status
+  verbs:
+    - get
+    - list
+    - watch
+- apiGroups:
+    - wgpolicyk8s.io
+  resources:
+    - policyreports
+    - clusterpolicyreports
+  verbs:
+    - create
+    - delete
+    - get
+    - list
+    - patch
+    - update
+    - watch
+- apiGroups:
+    - '*'
+  resources:
+    - '*'
+  verbs:
+    - get
+    - list
+    - watch
```

Comment:
I was thinking we should also give blank access to `apps/v1` (for `Deployment` and friends) and `networking.k8s.io/v1` (for Ingress). But then I realized that probably it's not a good idea to give access to confidential resources like `Secret`.

I propose to not define any new Role, but instead bind against the cluster role [`system:public-info-viewer`](https://kubernetes.io/docs/reference/access-authn-authz/rbac/).

Quoting the docs:

> Allows read-only access to non-sensitive information about the cluster. Introduced in Kubernetes v1.14.

Reply:
Sorry, I mean the `view` role, you can find it close to the one I mentioned above

Reply:
Done in https://github.com/kubewarden/helm-charts/pull/238/commits/2dd514092805371b50dea4632c545eb7fb762815.